include/rive/artboard.hpp

```diff
@@ -88,7 +92,7 @@ namespace rive {
 
         // Returns true iff calling popMessage() will return true.
         bool hasMessages() const;
-        
```

Comment:
void postPointerEvent(const PointerEvent&);

Should we remove this, now that you've (correctly) put the pointer posters on StateMachineInstance? (seems like it)

Reply:
removed